relay-server/src/actors/outcome.rs

```diff
@@ -84,7 +84,7 @@ pub struct TrackOutcome {
     /// The event's data category.
     pub category: DataCategory,
     /// The number of events or total attachment size in bytes.
-    pub quantity: usize,
+    pub quantity: u32,
```

Comment:
I'm changing this to u32 because we do not actually permit anything larger than a u32 on the kafka topic or it will fatally fail in the consumer.